src/mustache/Context.hx

```diff
@@ -80,7 +80,13 @@ private class ContextImpl {
     }
 
     static function getField(object:Dynamic, name:String):haxe.ds.Option<Dynamic> {
-        var value = Reflect.field(object, name);
+        var value;
```

Comment:
No need to declare this var, just move the whole `var value = Reflect.field(object, name)` expression after the check